taichi/backends/metal/taichi_file_util.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
```

Comment:
nit: Place `BufferSize` inside `kernel_utils.h`, then rename this to `aot_utils.h`?